src/redux/reducers/pretender/applications.js

```diff
@@ -0,0 +1,63 @@
+import { getApplicationsSuccess } from './mocks';
+
+export default function () {
+    this.get('/applications', (request) => {
```

Comment:
This is the file when you can mock all of the APIs calls. You can put any endpoint, and return what you need. Keep in mind that APIs usually return a JSON object. You can add some logic to make the call failed or succeed It is best to have one file per endpoint, it is easier to read.